Assets/Scripts/Gamemode/RacingGamemode.cs

```diff
@@ -22,10 +37,8 @@ public override void OnGameWon()
 
             var parameters = new object[] { 0 };
             FengGameManagerMKII.instance.photonView.RPC("netGameWin", PhotonTargets.Others, parameters);
-            if (((int) FengGameManagerMKII.settings[0xf4]) == 1)
-            {
-                //this.chatRoom.addLINE("<color=#FFC000>(" + this.roundTime.ToString("F2") + ")</color> Round ended (game win).");
-            }
+
+            //there was an if on (int) FengGameManagerMKII.settings[0xf4]) == 1 and just a comment inside
```

Comment:
Useless code, so good to remove that, but remove your comment as well then

Reply:
right